builtin/plugins/dposv3/test_helpers.go

```diff
@@ -349,6 +349,14 @@ func (dpos *testDPOSContract) Unbond(ctx *plugin.FakeContext, validator *loom.Ad
 	return err
 }
 
+func (dpos *testDPOSContract) UnbondAll(ctx *plugin.FakeContext) error {
```

Comment:
I don't think this abstraction is worth it, just makes the test harder to follow.